esx_service/vmdk_ops.py

```diff
@@ -294,16 +318,17 @@ def formatVmdk(vmdk_path, vol_name):
             return err(
                 "Unable to format %s and unable to delete volume. Please delete it manually."
                 % vmdk_path)
+    cleanupVSANDevFSNode(vmdk_path)
```

Comment:
Is there any harm in doing this independent of return code from MKFS command? Otherwise there is still possibility of leak if mkfs failed internally with genuine error.